### DISCUSSION TITLE
Error message: object 'Matrix_validate' not found

### DISCUSSION THREAD

**Discussion by LauHerr** [satijalab/seurat#6401] [Q&A]
Dear Seurat experts,
today I installed R 4.2.1, RStudio-2022.07.1-554 on a new machine running Windows 11. 
After running my code which was working before:

_**library(Seurat)
data_dir <- 'H:/LH/Group Folder/Single cell sequencing/XXXXX/XXXXX/'
list.files(data_dir)
expression_matrix <- Read10X(data.dir = data_dir)**_

 I get the following error message:

_**Error in validityMethod(as(object, superClass)) : 
  object 'Matrix_validate' not found**_

The exact same code is running on another machine completely fine.
I hope someone of you can offer me some help

Thank you very much in advance :)
Laurenz

**Comment by slamcole** [comment]
Hi Laurenz, did you get any help with the error?

I used the following 
library(Seurat)
setwd("F:/XXXXXXXXXXX/Single cell data")
data <- Read10X(data.dir ="R-analysis")

And got the error below
Error in validityMethod(as(object, superClass)) :
object 'Matrix_validate' not found

**Reply by LauHerr** [reply]
Hi Slamcole,
no, unfortunately, there wasn't any suggestion yet and I myself was also not able to fix the problem.

Greetings
Laurenz

**Reply by aion111** [reply]
Hello guys,

install.packages("Matrix") can solve this error.

**Reply by LauHerr** [reply]
Thanks a lot, that did the trick :)

**Comment by mariakondili** [comment]
Hello, 
Any chance the error  : `"Dim_validate" not found `, is also associated to this package ? But I have the latest Matrix v.1.6 already !  
I get the error after the function :  `FindIntegrationAnchors()` , where I try to integrate multiple samples of same condition as a seurat_list.
